API Blueprint Specification.md

```diff
@@ -478,6 +488,189 @@ Name and description of the API
 
 ---
 
+<a name="def-authentication-section"></a>
+## Authentication section
+- **Parent sections:** none, [Action section](#def-action-section), [Request section](#def-request-section)
+- **Nested sections:** [`1` Authentication schemes section](#def-authenticationschemes-section), [`1` Response section](#def-response-section)
+- **Markdown entity:** header, list
+- **Inherits from**: none
+
+#### Definition
+Defined by the `Authentication` keyword in Markdown header entity.
+
+    # Authentication
+
+**-- or --**
+
+Defined by the `Authentication` keyword in Markdown list entity followed by optional comma seperated list of scheme names which is prefixed by a colon.
+
+    + Authentication: <list>
+
+> **NOTE:** The second notation is only used when the parent section is either an [Action section](#def-action-section) or a [Request section](#def-request-section).
+
+#### Description
+Based on the parent section, the usage of this section differs as shown below.
```

Comment:
Trying to figure out what is the purpose of the section: Reading "authentication settings" vs. "authentication support" 

Consider what you have there: 

- Description of the authentication support for the whole API
- Description of the request level authentication settings
- Description of the action level authentication settings

Please stick with one (support vs settings) – I am in favor of the settings or perhaps another word (mechanism?)